test/test_nn.py

```diff
@@ -275,6 +275,14 @@ def _do_test(self, test_case, module, input):
 
             module_ip = self.constructor(*self.constructor_args, inplace=True)
 
+            # There is no guarantee that `input`'s version is zero at this point,
+            # so we manually bump `input`'s version to a much higher value, so that
+            # the subsequent version checks won't find `input_ip_clone`'s version
+            # and `input`'s version to be accidentally equal.
+            for i in range(20):
+                with torch.no_grad():
+                    input.add_(1)
```

Comment:
This is a nice catch.

However, assuming that what we are trying to determine is that `input_ip_clone` was in-place mutated, wouldn't it be easier to just compare it against what its version was before we called `module_ip` on it? (Or is the test trying to assert that the version counters are not shared? That's easy enough to test too: additionally check that input version didn't bump.)